2200036/04.py

```diff
@@ -0,0 +1,30 @@
+# 퍼펙트 셔플
+
+T = int(input())
+cards = int(input())
+for t in range(1, T + 1):
+    li = input().split()
+    shuffle = []
```

Comment:
```suggestion
T = int(input())
for t in range(1, T + 1):
    cards = int(input())
    li = input().split()
    shuffle = []
```
`for`문 안에서 입력을 받아주셔야 합니다!